dev-tools/validation/check-authz-route-matrix.py

```
@@ -134,42 +2476,212 @@ def validate_known_drift(value: Any, field: str, errors: list[str]) -> None:
+    if missing_required_routes:
+        errors.append(
+            "operator_mutation_support_gate.applies_to is missing required current "
+            f"session lifecycle routes: {missing_required_routes}"
+        )
+    for identity in sorted(
+        set(gate_identities) - route_identities - set(drift_identities)
+    ):
+        errors.append(
+            "operator_mutation_support_gate identity is neither a route nor explicit "
+            f"coverage drift: {identity}"
+        )
```

Ruff (0.16.0)

[warning] Use `list.extend` to create a transformed list

(PERF401)